### PR TITLE
Added option to disable telemetry and notice in readme

### PR DESCRIPTION
**Pull request by praveendath92**
Added notice about data collection and option to disable telemetry collection

### CODE REVIEW

**Comment by praveendath92** [comment]
@beatrizz here are the final changes for now

File: src/apkAnalyzer/CiRunner.ts [hunk 1]
```diff
@@ -42,12 +42,26 @@ export interface CiCore {
...
     constructor(ciCore: CiCore) {
         this.ciCore = ciCore;
         this.thresholdChecker = new ThresholdChecker(ciCore);
+    }
+
+    /**
+     * Runs the ci runner with telemetry enabled if set     
+     */
+    public async run() {
+        const telemtryEnabled = this.ciCore.getInput('telemetryEnabled');
```
Comment (beatrizz) [inline src/apkAnalyzer/CiRunner.ts]: typo in telemtry

File: src/apkAnalyzer/CiRunner.ts [hunk 2]
```diff
@@ -42,12 +42,26 @@ export interface CiCore {
...
 
+        if (telemtryEnabled != 'false') {
```
Comment (beatrizz) [inline src/apkAnalyzer/CiRunner.ts]: Can we do "if (!telemetryEnabled)"? (or "if (telemetryEnabled)" and reverse the statements)

Comment (praveendath92) [reply src/apkAnalyzer/CiRunner.ts]: Done

File: src/apkAnalyzer/CiRunner.ts (hunk 1 above)
Comment (praveendath92) [reply src/apkAnalyzer/CiRunner.ts]: Fixed

**Pull request merged**